bliss/simulator/prior.py

```diff
@@ -9,6 +9,33 @@
 
 from bliss.catalog import SourceType, TileCatalog
 
+PriorConfig = TypedDict(
```

Comment:
What's the benefit of having both a `PriorConfig` and a `CatalogPrior`? It seems simpler to only have a `CatalogPrior`.

Reply:
This is for static type-checking, and mainly because I added an extra `survey_bands` parameter to the `CatalogPrior::__init__`. It can be removed; it's just a nice-to-have

Reply:
I worry that future developers will get confused about their being two distinct types of prior-related objects, and I don't see a major benefit. Better to merge them, I think.

Reply:
Done.